up-l3/udiscovery/v3/service.adoc

```diff
@@ -0,0 +1,253 @@
+= uDiscovery Service
+:toc: preamble
+:sectnums:
+:source-highlighter: highlight.js
+:client-proto-ref: xref:../../../up-core-api/uprotocol/core/udiscovery/v3/udiscovery.proto[udiscovery.proto]
+:replicator-proto-ref: xref:../../../up-core-api/uprotocol/core/udiscovery/v3/udiscovery.proto[udiscovery_replicator.proto]
```

Comment:
you should only use `xref` for references to AsciiDoc resources. For everything else, use `link`